agilent_34401A_dmm.py

```diff
@@ -76,7 +90,51 @@ def read_voltage(self, c):
         set to voltage mode with configure_voltage. """
         dev = self.selectedDevice(c)
         ans = yield dev.query("READ?")
-        returnValue(float(ans))
+        returnValue(float(ans) * V)
+
+
+    @setting(15, AC = 'b', ac_highpass='v[Hz]', cRange='v[A]',
+             resolution ='v[V]')
+    def configure_current(self, c, AC=False, ac_highpass=3*Hz,
+                          cRange='AUTO', resolution=0.0001):
+        """ Sets the DMM to current mode, with given range and resolution. """
```

Comment:
Also resolution default value is not v[V], and it doesn't appear to be used in the function.